### ISSUE TITLE
Update Wiki

### ISSUE THREAD

**Issue by raymondcarl** [pqaidevteam/pqai#57]
Hi,

I wanted to fix this myself and submit a Pull Request, but apparently, you can't do it unless you are a added as a contributor to the project.

Looks like requirements.txt was removed and the dependencies were setup using Poetry.

On the local deployment wiki, I wanted to remove the references to requirements.txt and replace it with these steps:

cd pqai
sudo apt install pipx
pipx install poetry

I had some errors with the poetry lock file. I tried several steps, but just ended up deleting it. Regenerating it still caused the same error:
The lock file is not compatible with the current version of Poetry.
  Upgrade Poetry to be able to read the lock file or, alternatively, regenerate the lock file with the `poetry lock` command.

  at /usr/lib/python3/dist-packages/poetry/packages/locker.py:481 in _get_lock_data
      477│                 "Upgrade Poetry to ensure the lock file is read properly or, alternatively, "
      478│                 "regenerate the lock file with the `poetry lock` command."
      479│             )
      480│         elif not lock_version_allowed:
    → 481│             raise RuntimeError(
      482│                 "The lock file is not compatible with the current version of Poetry.\n"
      483│                 "Upgrade Poetry to be able to read the lock file or, alternatively, "
      484│                 "regenerate the lock file with the `poetry lock` command."
      485│             )

I found it easier to just delete the lock file.

**Comment by mahesh-maan** [comment]
Hi @raymondcarl thanks for pointing this out. I have updated the deployment instruction on Wiki. It looks like that the issue you faced could have been due to a different Poetry version, so I've mentioned the version I used in the instructions.

Also, I didn't realize that by default Wikis can't be edited by non-contributors. There is a setting for that - I have flipped it now. So any Github user can make edits now.
